src/modules/Home/hooks/useProjectList.ts

```diff
@@ -0,0 +1,9 @@
+import React, { useState } from 'react';
+import { ProjectData } from '@/types/projects';
+
+export function useProjectList(
+  projects?: ProjectData[],
+): [ProjectData[] | [], React.Dispatch<ProjectData[] | []>] {
```

Comment:
It's not required to defined an empty array
```suggestion
): [ProjectData[], React.Dispatch<ProjectData[]>] {
```
Or did you do it because of other specific reasons?